microsoft-edge/devtools-guide-chromium/memory-problems/memory-101.md

```diff
@@ -23,222 +23,175 @@ ms.date: 07/21/2023
+_Garbage-collection roots_ (_GC roots_) are created by the browser when a reference is made from the browser's native code to a JavaScript object that's outside of the V8 virtual machine. These references are called _handles_.
 
-*  The document DOM tree, consisting of all native DOM nodes that are reachable by traversing the document.  Not all of the nodes have JavaScript wrappers, but if a node has a wrapper, the node is alive while the document is alive.
+There are many internal GC roots, most of which aren't interesting for web developers. From the webpage's standpoint, the following types of GC roots exist:
 
-*  Sometimes objects are retained by the debug context in the **Sources** tool and the **Console**, such as after Console evaluation.  Create heap snapshots with a cleared **Console** tool and no active breakpoints in the debugger in the **Sources** tool.
+*  Window global objects (one in each iframe).
 
->[!TIP]
-> Before taking a heap snapshot in the [Memory](heap-snapshots.md) tool, clear the **Console** tool and deactivate breakpoints in the **Sources** tool.  To clear the **Console** tool, run the `clear()` method.
+*  Sometimes objects are retained by the debugging context that's set by the **Sources** or the **Console** tool, such as when evaluating a JavaScript expression in the **Console** tool. To remove these objects from the **Memory** tool, before recording a heap snapshot, clear the **Console** tool and deactivate breakpoints in the **Sources** tool.
 
-The memory graph starts with a root, which may be the `window` object of the browser or the `Global` object of a Node.js module.  You don't control how the root object is garbage-collected.
+The memory graph starts with a GC root, which may be the `window` object of the browser or the `Global` object of a Node.js module. You don't control how the root object is garbage-collected:
 
 ![You can't control how the root object is garbage-collected](./memory-101-images/dontcontrol.png)
 
-Whatever isn't reachable from the root gets garbage-collected.
-
-> [!NOTE]
-> The number that's shown in the [Shallow size](#shallow-size) and [Retained size](#retained-size) columns is the number of bytes.
+Nodes that aren't reachable from the root can get garbage-collected.
 
 
 <!-- ====================================================================== -->
-## Objects retaining tree
-
-The heap is a network of interconnected objects.  In the mathematical world, this structure is called a _graph_ or _memory graph_.  A graph is constructed from _nodes_ that are connected by _edges_.
+## Object sizes and distances
 
-Nodes and edges in a graph are given labels as follows:
+When working with the **Memory** tool, you will likely find yourself looking at the following columns of information:
 
-*  _Nodes_ (or _objects_) are labelled using the name of the _constructor_ function that was used to build them.
+* **Shallow Size**
+* **Retained Size**
+* **Distance**
```

Comment:
```suggestion
* **Distance**
* **Shallow Size**
* **Retained Size**
```
same order as in png 
rendered:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/memory-problems/memory-101?branch=pr-en-us-3033#object-sizes-and-distances